.github/workflows/build-extensions.yml

```diff
@@ -150,11 +150,11 @@ jobs:
         include:
           - name: Plugins_g++
             compiler: g++
-            docker_tag: ubuntu-build-gcc
+            docker_tag: ubuntu-build-opencv-gcc
```

Comment:
I prefer `ubuntu-build-gcc-plugins-deps`. Or any unified name instead of using `opencv` only.

Reply:
And we should have separate PR.
One is for adding new docker images.
Another is for updating the workflows.
The reason is that we are not allow a PR to update docker hub unless it gets merged for the security issue.